test/jdk/java/lang/constant/MethodHandleDescTest.java

```diff
@@ -359,7 +361,21 @@ public void testSymbolicDescsConstants() throws ReflectiveOperationException {
 
     public void testKind() {
         for (Kind k : Kind.values()) {
+            assertEquals(Kind.valueOf(k.refKind), Kind.valueOf(k.refKind, k.refKind == MethodHandleInfo.REF_invokeInterface));
             assertEquals(Kind.valueOf(k.refKind, k.isInterface), k);
         }
+        // let's now verify those cases for which the value of the isInterface parameter is ignored
+        int[] isInterfaceIgnored = new int[] {
+                REF_getField,
+                REF_getStatic,
+                REF_putField,
+                REF_putStatic,
+                REF_newInvokeSpecial,
+                REF_invokeInterface
+        };
+
+        for (int refKind : isInterfaceIgnored) {
+            assertEquals(Kind.valueOf(refKind, false), Kind.valueOf(refKind, true));
+        }
```

Comment:
Can you also add the test cases to verify `Kind::valueOf` with `REF_invokeStatic` and `REF_invokeSpecial`?

Reply:
please see them below, thanks